packages/shared/components/ColorPicker.svelte

```diff
@@ -1,118 +1,146 @@
-            on:click={activeCustomColor}
-            on:mouseenter={toggleCustomHover}
-            on:mouseleave={toggleCustomHover}
-        >
-            <Icon icon="edit" classes={isCustomHover ? 'text-gray-800' : `text-${inputColor}`} />
-        </li>
+        {#if isCustomColorEnabled}
+            <li
+                bind:this={tooltipAnchor}
+                on:click={toggleTooltip}
+                on:click={activeCustomColor}
+                on:keypress={toggleTooltip}
+                on:keypress={activeCustomColor}
+                on:mouseenter={toggleCustomHover}
+                on:mouseleave={toggleCustomHover}
+                tabindex="0"
+                class="w-12 h-12 rounded-lg ring-opacity-30 hover:ring-opacity-40 cursor-pointer flex justify-center items-center
+                custom-color hover:bg-gray-50 focus:bg-white ring-white"
+                class:active={customActiveFilled}
+                class:ring-4={customActiveFilled}
+            >
+                <Icon icon={IconEnum.Edit} classes={isCustomHover ? 'text-gray-800' : `text-${inputColor}`} />
+            </li>
+        {/if}
     </ul>
-    {#if showTooltip}
-        <div use:clickOutside on:clickOutside={toggleTooltip}>
-            <Tooltip anchor={tooltipAnchor} position="top">
-                <Text type="p">{localize('views.picker.color.hexCode')}</Text>
+    {#if isTooltipVisible}
+        <tooltip-container class="block" use:clickOutside on:clickOutside={toggleTooltip}>
+            <Tooltip anchor={tooltipAnchor} position={Position.Top}>
+                <Text type={TextType.p}>{localize('views.picker.color.hexCode')}</Text>
                 <input
+                    bind:value={inputValue}
+                    on:click={activeCustomColor}
                     type="text"
                     placeholder="#"
                     pattern="[A-F0-9]{10}"
                     maxlength="7"
-                    bind:value={inputValue}
-                    on:click={activeCustomColor}
-                    class="w-24 h-full text-16 uppercase leading-140 border border-solid mt-2 bg-white dark:bg-gray-800 border-gray-300
-                    text-gray-800 dark:text-white dark:border-gray-700
-                    hover:border-gray-500 dark:hover:border-gray-700 p-1 rounded text-center"
                     class:ring-4={customActiveFilled}
                     class:active={customActiveFilled}
                 />
             </Tooltip>
-        </div>
+        </tooltip-container>
     {/if}
-</div>
+</color-picker>
 
-<style type="text/scss">
+<style lang="scss">
     input {
-        background-color: transparent;
+        @apply w-24 h-full mt-2 p-1 rounded;
+        @apply uppercase text-16 leading-140 text-gray-800 dark:text-white text-center;
+        @apply border border-solid border-gray-300 dark:border-gray-700 hover:border-gray-500 dark:hover:border-gray-700;
+        @apply bg-transparent dark:bg-gray-800;
```

Comment:
all the `dark:` variants here cause the app to crash, this isnt allowed in tailwind 2